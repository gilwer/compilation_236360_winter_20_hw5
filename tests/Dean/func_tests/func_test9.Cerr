[[[ added func: print with 1 params and label: ]]]
[[[ added func: printi with 1 params and label: ]]]
print param:2(1)
printi param:3(1)
[FIRST_PROGRAM_POINT]
[set_func_label] label print for function print
[set_func_label] label printi for function printi
(END_OF)  [FIRST_PROGRAM_POINT]
