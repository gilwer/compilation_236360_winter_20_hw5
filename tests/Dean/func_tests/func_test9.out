Loaded: ./exceptions.s
32
